portal/src/content/docs/documentation/working-with-tables.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Working With Tables
+sidebar:
+  order: 2
+---
+
+This section explains about how to work with tabular data in open data editor. Tabular data is structured as rows and columns, and each row represents an instance of the data and each column represents an attribute of that data.
+
+## Adding and Opening Table
+
+If you have a tabular data file for example table.csv then you can open the file by clicking table name in "File Explorer" on the left side of the editor. It supports other formats also, such as csv, xls, parq, ods etc.
+
+![ADD TABLE](../../../assets/table.png)
```

Comment:
Can you please try moving table-related assets to the same folder as an article?

- documentation
  - working-with-tables
    - assets
    - index.md

When we have more articles it will be easier to navigate I think